libs/features/mas/src/sidenav/merch-sidenav.js

```diff
@@ -29,6 +29,10 @@ export class MerchSideNav extends LitElement {
                 z-index: 2;
             }
 
+            :host h2 {
```

Comment:
looks liks some issue with rebase, i assume this change doesn't belong to this PR

Reply:
Yeah, issue with rebase. I only modified lana.js and its unit tests. I don't see that in the PR any longer, unless i missed something.